e2e_tests.cloudbuild.yaml

```diff
@@ -0,0 +1,84 @@
+steps:
+  - id: Setup Script
+    name: 'google/cloud-sdk'
+    entrypoint: /bin/bash
+    args:
+      - '-c'
+      - |
+        pip3 install --upgrade pip
+        cd setup/
+        export FOURKEYS_PROJECT=${PROJECT_ID}
+        printf "n\ny\n2\n1\nn\ny\n" | ./setup.sh
```

Comment:
This works, but is less readable.  Potentially we should save this kind of information from a run of setup.sh and import it.

Reply:
Also, it would be great to do some comment explanations here.

Reply:
Added comments -- is it clearer?